src/Playwright/Page.cs

```diff
@@ -94,7 +94,17 @@ internal Page(IChannelOwner parent, string guid, PageInitializer initializer) :
             _channel.Route += Channel_Route;
             _channel.FrameAttached += Channel_FrameAttached;
             _channel.FrameDetached += Channel_FrameDetached;
-            _channel.Dialog += (_, e) => Dialog?.Invoke(this, e);
+            _channel.Dialog += (_, e) =>
+            {
+                if (Dialog == null)
+                {
+                    _ = e.DismissAsync().ConfigureAwait(false);
```

Comment:
No need of a `ConfigureAwait` without an `await` :)
```suggestion
                    _ = e.DismissAsync();
```